pkg/csi/manager.go

```diff
@@ -54,6 +55,12 @@ func (m *Manager) Run(cfg *config.Config) error {
 		return err
 	}
 
+	storageClient, err := storage.NewFactoryFromConfig(restConfig)
+	if err != nil {
+		logrus.Fatalf("new storage client fail with err %v", err)
```

Comment:
This might be duplicated with the returned error.

Reply:
removed, thanks!

Reply:
Removed!